pkg/multus/multus.go

```diff
@@ -815,8 +815,13 @@ func CmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) er
 	}
 
 	if in.ReadinessIndicatorFile != "" {
-		if err := types.GetReadinessIndicatorFile(in.ReadinessIndicatorFile); err != nil {
-			return cmdErr(k8sArgs, "PollImmediate error waiting for ReadinessIndicatorFile (on del): %v", err)
+		readinessfileexists, err := types.ReadinessIndicatorExistsNow(in.ReadinessIndicatorFile)
+		if err != nil {
+			return cmdErr(k8sArgs, "error checking readinessindicatorfile on CNI DEL @ %v: %v", in.ReadinessIndicatorFile, err)
+		} else {
```

Comment:
As of test warning message in https://github.com/k8snetworkplumbingwg/multus-cni/pull/1235/files#annotation_18379130107 you can remove else block (because if clause ends with return, hence we don't need else block.